lib/Controller/RoomController.php

```diff
@@ -1109,7 +1110,7 @@ public function getParticipants(bool $includeStatus = false): DataResponse {
 		}
 
 		$maxPingAge = $this->timeFactory->getTime() - 100;
-		$participants = $this->participantService->getParticipantsForRoom($this->room);
+		$participants = $this->participantService->getParticipantsForRoom($this->room, true); // FIXME should return some what session info?
```

Comment:
This here however is more complicated. We want to show online-state and call icons, but it basically means we would need to select all sessions and then group by actor. We can of course do that, and return a combined call flag of all sessions, but which session id should be returned?
I think currently some clients abuse this for their call screen instead of `CallController::getPeersForCall` (even the web?).

From my POV this should be fixed to use the proper API and then the clients when receiving "please refresh participant list" they refresh participants and peer list for the internal signaling and refresh participants for external signaling + use the data of the event to update their peers.